sources/include/citygml/attributesmap.h

```diff
@@ -52,12 +52,19 @@ class LIBCITYGML_EXPORT AttributeValue
     double asDouble(double defaultValue=0.0) const;
     int asInteger(int defaultValue=0) const;
     AttributesMap& asAttributeSet();
-    const AttributesMap& asAttributeSet() const;
+    [[nodiscard]] const AttributesMap& asAttributeSet() const;
+
+    // Returns formatted string of AttributesMap, includes recursive children.
+    static std::string attributesMapToString(const AttributesMap& attributesMap);
+
 private:
     AttributeType m_type;
     std::string m_value;
     // Avoid using string field because of parse latency issue.
     AttributesMap m_attribute_set;
+
+    static std::string attributesMapToStringRecursive(const AttributesMap &attributesMap, int depth);
+    static std::string indent(int num);
 };
 
 LIBCITYGML_EXPORT std::ostream& operator<<(std::ostream& os, const AttributeValue& o);
```

Comment:
AttributesMapのstreamのオーバーロードとして実装お願いします。

```suggestion
LIBCITYGML_EXPORT std::ostream& operator<<(std::ostream& os, const AttributeValue& o);
LIBCITYGML_EXPORT std::ostream& operator<<(std::ostream& os, const AttributesMap& o);
```

Reply:
修正しました！